src/lib.rs

```diff
@@ -27,4 +27,5 @@ pub mod script;
 
 // Re-export tari_utils
 pub use hash::blake2 as common;
+pub use signature::schnorr as signatures;
```

Comment:
disagree with this, rather just make a breaking change or expose it normally.

How about:
rename `pub mod signature` to `pub mod signatures`
and add
`pub use schorr::*` in `signatures.rs`